internal/btf/btf.go

```diff
@@ -451,13 +455,6 @@ func ProgramLineInfos(s *Program) (recordSize uint32, bytes []byte, err error) {
 	return s.lineInfos.recordSize, bytes, nil
 }
 
-// IsNotSupported returns true if the error indicates that the kernel
-// doesn't support BTF.
-func IsNotSupported(err error) bool {
-	var ufe *internal.UnsupportedFeatureError
-	return xerrors.As(err, &ufe) && ufe.Name == "BTF"
```

Comment:
i'm not quite sure, why `ufe.Name == "BTF"` was introduced. Please correct me, but I think, it should be fine without this second check.

Reply:
I was trying to be extra cautious, it doesn't make a difference for the code at the moment.